--- gst-plugins-base-1.14.5.old/common/glib-gen.mak	2018-03-23 20:44:44.000000000 -0000
+++ gst-plugins-base-1.14.5/common/glib-gen.mak	2020-01-25 02:40:57.375758050 -0000
@@ -1,11 +1,13 @@
 # these are the variables your Makefile.am should set
 # the example is based on the colorbalance interface
 
+H := \#
+
 #glib_enum_headers=$(colorbalance_headers)
 #glib_enum_define=GST_COLOR_BALANCE
 #glib_enum_prefix=gst_color_balance
 
-enum_headers=$(foreach h,$(glib_enum_headers),\n\#include \"$(h)\")
+enum_headers=$(foreach h,$(glib_enum_headers),\n$(H)include \"$(h)\")
 
 # these are all the rules generating the relevant files
 %-marshal.h: %-marshal.list
diff -Naur gst-plugins-base-1.14.5.old/common/gst-glib-gen.mak gst-plugins-base-1.14.5/common/gst-glib-gen.mak
--- gst-plugins-base-1.14.5.old/common/gst-glib-gen.mak	2018-03-23 20:44:44.000000000 -0000
+++ gst-plugins-base-1.14.5/common/gst-glib-gen.mak	2020-01-25 02:40:57.376758050 -0000
@@ -1,14 +1,16 @@
 # these are the variables your Makefile.am should set
 # the example is based on the colorbalance interface
 
+H := \#
+
 #glib_enum_headers=$(colorbalance_headers)
 #glib_enum_define=GST_COLOR_BALANCE
 #glib_gen_prefix=gst_color_balance
 #glib_gen_basename=colorbalance
 #glib_gen_decl_banner=GST_EXPORT
-#glib_gen_decl_include=\#include <gst/foo/foo-prelude.h>
+#glib_gen_decl_include=$(H)include <gst/foo/foo-prelude.h>
 
-enum_headers=$(foreach h,$(glib_enum_headers),\n\#include \"$(h)\")
+enum_headers=$(foreach h,$(glib_enum_headers),\n$(H)include \"$(h)\")
 
 # these are all the rules generating the relevant files
 $(glib_gen_basename)-marshal.h: $(glib_gen_basename)-marshal.list
diff -Naur gst-plugins-base-1.14.5.old/gst/encoding/Makefile.in gst-plugins-base-1.14.5/gst/encoding/Makefile.in
--- gst-plugins-base-1.14.5.old/gst/encoding/Makefile.in	2019-05-29 16:33:24.000000000 -0000
+++ gst-plugins-base-1.14.5/gst/encoding/Makefile.in	2020-01-25 02:42:51.666748132 -0000
@@ -17,12 +17,14 @@
 # these are the variables your Makefile.am should set
 # the example is based on the colorbalance interface
 
+H := \#
+
 #glib_enum_headers=$(colorbalance_headers)
 #glib_enum_define=GST_COLOR_BALANCE
 #glib_gen_prefix=gst_color_balance
 #glib_gen_basename=colorbalance
 #glib_gen_decl_banner=GST_EXPORT
-#glib_gen_decl_include=\#include <gst/foo/foo-prelude.h>
+#glib_gen_decl_include=$(H)include <gst/foo/foo-prelude.h>
 
 
 VPATH = @srcdir@
@@ -638,7 +640,7 @@
 	gstsmartencoder.h	\
 	gststreamcombinerpad.h
 
-enum_headers = $(foreach h,$(glib_enum_headers),\n\#include \"$(h)\")
+enum_headers = $(foreach h,$(glib_enum_headers),\n$(H)include \"$(h)\")
 all: all-am
 
 .SUFFIXES:
diff -Naur gst-plugins-base-1.14.5.old/gst-libs/gst/app/Makefile.am gst-plugins-base-1.14.5/gst-libs/gst/app/Makefile.am
--- gst-plugins-base-1.14.5.old/gst-libs/gst/app/Makefile.am	2018-03-23 20:44:42.000000000 -0000
+++ gst-plugins-base-1.14.5/gst-libs/gst/app/Makefile.am	2020-01-25 02:40:57.376758050 -0000
@@ -1,10 +1,12 @@
 lib_LTLIBRARIES = libgstapp-@GST_API_VERSION@.la
 
+H := \#
+
 glib_enum_define = GST_APP
 glib_gen_prefix = __gst_app
 glib_gen_basename = app
 glib_gen_decl_banner=GST_APP_API
-glib_gen_decl_include=\#include <gst/app/app-prelude.h>
+glib_gen_decl_include=$(H)include <gst/app/app-prelude.h>
 
 glib_enum_headers = gstappsrc.h
 
diff -Naur gst-plugins-base-1.14.5.old/gst-libs/gst/app/Makefile.in gst-plugins-base-1.14.5/gst-libs/gst/app/Makefile.in
--- gst-plugins-base-1.14.5.old/gst-libs/gst/app/Makefile.in	2019-05-29 16:33:22.000000000 -0000
+++ gst-plugins-base-1.14.5/gst-libs/gst/app/Makefile.in	2020-01-25 02:42:51.667748132 -0000
@@ -17,12 +17,14 @@
 # these are the variables your Makefile.am should set
 # the example is based on the colorbalance interface
 
+H := \#
+
 #glib_enum_headers=$(colorbalance_headers)
 #glib_enum_define=GST_COLOR_BALANCE
 #glib_gen_prefix=gst_color_balance
 #glib_gen_basename=colorbalance
 #glib_gen_decl_banner=GST_EXPORT
-#glib_gen_decl_include=\#include <gst/foo/foo-prelude.h>
+#glib_gen_decl_include=$(H)include <gst/foo/foo-prelude.h>
 
 
 
@@ -618,12 +620,12 @@
 glib_gen_prefix = __gst_app
 glib_gen_basename = app
 glib_gen_decl_banner = GST_APP_API
-glib_gen_decl_include = \#include <gst/app/app-prelude.h>
+glib_gen_decl_include = $(H)include <gst/app/app-prelude.h>
 glib_enum_headers = gstappsrc.h
 built_sources = app-enumtypes.c
 built_headers = app-enumtypes.h
 BUILT_SOURCES = $(built_sources) $(built_headers)
-enum_headers = $(foreach h,$(glib_enum_headers),\n\#include \"$(h)\")
+enum_headers = $(foreach h,$(glib_enum_headers),\n$(H)include \"$(h)\")
 libgstapp_@GST_API_VERSION@_la_SOURCES = gstappsrc.c gstappsink.c
 nodist_libgstapp_@GST_API_VERSION@_la_SOURCES = $(BUILT_SOURCES)
 libgstapp_@GST_API_VERSION@_la_CFLAGS = $(GST_PLUGINS_BASE_CFLAGS) \
diff -Naur gst-plugins-base-1.14.5.old/gst-libs/gst/audio/Makefile.am gst-plugins-base-1.14.5/gst-libs/gst/audio/Makefile.am
--- gst-plugins-base-1.14.5.old/gst-libs/gst/audio/Makefile.am	2018-03-23 20:44:42.000000000 -0000
+++ gst-plugins-base-1.14.5/gst-libs/gst/audio/Makefile.am	2020-01-25 02:40:57.376758050 -0000
@@ -2,6 +2,8 @@
 ORC_SOURCE=gstaudiopack
 include $(top_srcdir)/common/orc.mak
 
+H := \#
+
 glib_enum_headers= 		\
 	audio.h			\
 	audio-format.h			\
@@ -20,7 +22,7 @@
 glib_gen_prefix = gst_audio
 glib_gen_basename = audio
 glib_gen_decl_banner=GST_AUDIO_API
-glib_gen_decl_include=\#include <gst/audio/audio-prelude.h>
+glib_gen_decl_include=$(H)include <gst/audio/audio-prelude.h>
 
 built_sources = audio-enumtypes.c
 built_headers = audio-enumtypes.h
diff -Naur gst-plugins-base-1.14.5.old/gst-libs/gst/audio/Makefile.in gst-plugins-base-1.14.5/gst-libs/gst/audio/Makefile.in
--- gst-plugins-base-1.14.5.old/gst-libs/gst/audio/Makefile.in	2019-05-29 16:33:23.000000000 -0000
+++ gst-plugins-base-1.14.5/gst-libs/gst/audio/Makefile.in	2020-01-25 02:42:51.685748131 -0000
@@ -39,12 +39,14 @@
 # these are the variables your Makefile.am should set
 # the example is based on the colorbalance interface
 
+H := \#
+
 #glib_enum_headers=$(colorbalance_headers)
 #glib_enum_define=GST_COLOR_BALANCE
 #glib_gen_prefix=gst_color_balance
 #glib_gen_basename=colorbalance
 #glib_gen_decl_banner=GST_EXPORT
-#glib_gen_decl_include=\#include <gst/foo/foo-prelude.h>
+#glib_gen_decl_include=$(H)include <gst/foo/foo-prelude.h>
 
 
 
@@ -784,7 +786,7 @@
 glib_gen_prefix = gst_audio
 glib_gen_basename = audio
 glib_gen_decl_banner = GST_AUDIO_API
-glib_gen_decl_include = \#include <gst/audio/audio-prelude.h>
+glib_gen_decl_include = $(H)include <gst/audio/audio-prelude.h>
 built_sources = audio-enumtypes.c
 built_headers = audio-enumtypes.h
 lib_LTLIBRARIES = \
@@ -895,7 +897,7 @@
 @HAVE_X86_TRUE@	$(GST_LIB_LDFLAGS) \
 @HAVE_X86_TRUE@	$(GST_ALL_LDFLAGS)
 
-enum_headers = $(foreach h,$(glib_enum_headers),\n\#include \"$(h)\")
+enum_headers = $(foreach h,$(glib_enum_headers),\n$(H)include \"$(h)\")
 
 # Introspection
 @HAVE_INTROSPECTION_TRUE@BUILT_GIRSOURCES = GstAudio-@GST_API_VERSION@.gir
diff -Naur gst-plugins-base-1.14.5.old/gst-libs/gst/pbutils/Makefile.am gst-plugins-base-1.14.5/gst-libs/gst/pbutils/Makefile.am
--- gst-plugins-base-1.14.5.old/gst-libs/gst/pbutils/Makefile.am	2018-03-23 20:44:42.000000000 -0000
+++ gst-plugins-base-1.14.5/gst-libs/gst/pbutils/Makefile.am	2020-01-25 02:40:57.377758050 -0000
@@ -12,13 +12,15 @@
 	gstdiscoverer.h \
 	gstaudiovisualizer.h
 
+H := \#
+
 # variables used for enum/marshal generation
 glib_enum_headers = $(headers_pbutils)
 glib_enum_define = PB_UTILS
 glib_gen_prefix = pbutils
 glib_gen_basename = pbutils
 glib_gen_decl_banner=GST_PBUTILS_API
-glib_gen_decl_include=\#include <gst/pbutils/pbutils-prelude.h>
+glib_gen_decl_include=$(H)include <gst/pbutils/pbutils-prelude.h>
 
 built_headers_configure = \
 	gstpluginsbaseversion.h
diff -Naur gst-plugins-base-1.14.5.old/gst-libs/gst/pbutils/Makefile.in gst-plugins-base-1.14.5/gst-libs/gst/pbutils/Makefile.in
--- gst-plugins-base-1.14.5.old/gst-libs/gst/pbutils/Makefile.in	2019-05-29 16:33:23.000000000 -0000
+++ gst-plugins-base-1.14.5/gst-libs/gst/pbutils/Makefile.in	2020-01-25 02:42:51.676748131 -0000
@@ -17,12 +17,14 @@
 # these are the variables your Makefile.am should set
 # the example is based on the colorbalance interface
 
+H := \#
+
 #glib_enum_headers=$(colorbalance_headers)
 #glib_enum_define=GST_COLOR_BALANCE
 #glib_gen_prefix=gst_color_balance
 #glib_gen_basename=colorbalance
 #glib_gen_decl_banner=GST_EXPORT
-#glib_gen_decl_include=\#include <gst/foo/foo-prelude.h>
+#glib_gen_decl_include=$(H)include <gst/foo/foo-prelude.h>
 
 
 
@@ -671,7 +673,7 @@
 glib_gen_prefix = pbutils
 glib_gen_basename = pbutils
 glib_gen_decl_banner = GST_PBUTILS_API
-glib_gen_decl_include = \#include <gst/pbutils/pbutils-prelude.h>
+glib_gen_decl_include = $(H)include <gst/pbutils/pbutils-prelude.h>
 built_headers_configure = \
 	gstpluginsbaseversion.h
 
@@ -719,7 +721,7 @@
 
 # DISTCLEANFILES is for files generated by configure
 DISTCLEANFILES = $(built_headers_configure)
-enum_headers = $(foreach h,$(glib_enum_headers),\n\#include \"$(h)\")
+enum_headers = $(foreach h,$(glib_enum_headers),\n$(H)include \"$(h)\")
 @HAVE_INTROSPECTION_TRUE@BUILT_GIRSOURCES = GstPbutils-@GST_API_VERSION@.gir
 @HAVE_INTROSPECTION_TRUE@gir_headers = $(patsubst %,$(srcdir)/%, \
 @HAVE_INTROSPECTION_TRUE@	$(libgstpbutils_@GST_API_VERSION@include_HEADERS)) \
diff -Naur gst-plugins-base-1.14.5.old/gst-libs/gst/rtp/Makefile.am gst-plugins-base-1.14.5/gst-libs/gst/rtp/Makefile.am
--- gst-plugins-base-1.14.5.old/gst-libs/gst/rtp/Makefile.am	2018-03-23 20:44:42.000000000 -0000
+++ gst-plugins-base-1.14.5/gst-libs/gst/rtp/Makefile.am	2020-01-25 02:40:57.377758050 -0000
@@ -35,12 +35,14 @@
 BUILT_SOURCES = $(built_headers) $(built_sources)
 CLEANFILES = $(BUILT_SOURCES)
 
+H := \#
+
 glib_gen_prefix = __gst_rtp
 glib_enum_define = gst_rtp
 glib_enum_headers = $(libgstrtpinclude_HEADERS)
 glib_gen_basename = gstrtp
 glib_gen_decl_banner=GST_RTP_API
-glib_gen_decl_include=\#include <gst/rtp/rtp-prelude.h>
+glib_gen_decl_include=$(H)include <gst/rtp/rtp-prelude.h>
 
 include $(top_srcdir)/common/gst-glib-gen.mak
 
diff -Naur gst-plugins-base-1.14.5.old/gst-libs/gst/rtp/Makefile.in gst-plugins-base-1.14.5/gst-libs/gst/rtp/Makefile.in
--- gst-plugins-base-1.14.5.old/gst-libs/gst/rtp/Makefile.in	2019-05-29 16:33:23.000000000 -0000
+++ gst-plugins-base-1.14.5/gst-libs/gst/rtp/Makefile.in	2020-01-25 02:42:51.672748132 -0000
@@ -17,12 +17,14 @@
 # these are the variables your Makefile.am should set
 # the example is based on the colorbalance interface
 
+H := \#
+
 #glib_enum_headers=$(colorbalance_headers)
 #glib_enum_define=GST_COLOR_BALANCE
 #glib_gen_prefix=gst_color_balance
 #glib_gen_basename=colorbalance
 #glib_gen_decl_banner=GST_EXPORT
-#glib_gen_decl_include=\#include <gst/foo/foo-prelude.h>
+#glib_gen_decl_include=$(H)include <gst/foo/foo-prelude.h>
 
 
 
@@ -675,8 +677,8 @@
 glib_enum_headers = $(libgstrtpinclude_HEADERS)
 glib_gen_basename = gstrtp
 glib_gen_decl_banner = GST_RTP_API
-glib_gen_decl_include = \#include <gst/rtp/rtp-prelude.h>
-enum_headers = $(foreach h,$(glib_enum_headers),\n\#include \"$(h)\")
+glib_gen_decl_include = $(H)include <gst/rtp/rtp-prelude.h>
+enum_headers = $(foreach h,$(glib_enum_headers),\n$(H)include \"$(h)\")
 @HAVE_INTROSPECTION_TRUE@BUILT_GIRSOURCES = GstRtp-@GST_API_VERSION@.gir
 @HAVE_INTROSPECTION_TRUE@gir_headers = $(patsubst %,$(srcdir)/%, \
 @HAVE_INTROSPECTION_TRUE@	$(libgstrtpinclude_HEADERS)) \
diff -Naur gst-plugins-base-1.14.5.old/gst-libs/gst/rtsp/Makefile.am gst-plugins-base-1.14.5/gst-libs/gst/rtsp/Makefile.am
--- gst-plugins-base-1.14.5.old/gst-libs/gst/rtsp/Makefile.am	2018-03-23 20:44:42.000000000 -0000
+++ gst-plugins-base-1.14.5/gst-libs/gst/rtsp/Makefile.am	2020-01-25 02:40:57.377758050 -0000
@@ -41,12 +41,14 @@
 BUILT_SOURCES = $(built_headers) $(built_sources)
 CLEANFILES = $(BUILT_SOURCES)
 
+H := \#
+
 glib_gen_prefix = __gst_rtsp
 glib_enum_define = gst_rtsp
 glib_enum_headers = $(libgstrtspinclude_HEADERS)
 glib_gen_basename = gstrtsp
 glib_gen_decl_banner=GST_RTSP_API
-glib_gen_decl_include=\#include <gst/rtsp/rtsp-prelude.h>
+glib_gen_decl_include=$(H)include <gst/rtsp/rtsp-prelude.h>
 
 include $(top_srcdir)/common/gst-glib-gen.mak
 
diff -Naur gst-plugins-base-1.14.5.old/gst-libs/gst/rtsp/Makefile.in gst-plugins-base-1.14.5/gst-libs/gst/rtsp/Makefile.in
--- gst-plugins-base-1.14.5.old/gst-libs/gst/rtsp/Makefile.in	2019-05-29 16:33:23.000000000 -0000
+++ gst-plugins-base-1.14.5/gst-libs/gst/rtsp/Makefile.in	2020-01-25 02:42:51.681748131 -0000
@@ -17,12 +17,14 @@
 # these are the variables your Makefile.am should set
 # the example is based on the colorbalance interface
 
+H := \#
+
 #glib_enum_headers=$(colorbalance_headers)
 #glib_enum_define=GST_COLOR_BALANCE
 #glib_gen_prefix=gst_color_balance
 #glib_gen_basename=colorbalance
 #glib_gen_decl_banner=GST_EXPORT
-#glib_gen_decl_include=\#include <gst/foo/foo-prelude.h>
+#glib_gen_decl_include=$(H)include <gst/foo/foo-prelude.h>
 
 
 
@@ -679,8 +681,8 @@
 glib_enum_headers = $(libgstrtspinclude_HEADERS)
 glib_gen_basename = gstrtsp
 glib_gen_decl_banner = GST_RTSP_API
-glib_gen_decl_include = \#include <gst/rtsp/rtsp-prelude.h>
-enum_headers = $(foreach h,$(glib_enum_headers),\n\#include \"$(h)\")
+glib_gen_decl_include = $(H)include <gst/rtsp/rtsp-prelude.h>
+enum_headers = $(foreach h,$(glib_enum_headers),\n$(H)include \"$(h)\")
 @HAVE_INTROSPECTION_TRUE@BUILT_GIRSOURCES = GstRtsp-@GST_API_VERSION@.gir
 @HAVE_INTROSPECTION_TRUE@gir_headers = $(patsubst %,$(srcdir)/%, \
 @HAVE_INTROSPECTION_TRUE@	$(libgstrtspinclude_HEADERS)) \
diff -Naur gst-plugins-base-1.14.5.old/gst-libs/gst/tag/Makefile.am gst-plugins-base-1.14.5/gst-libs/gst/tag/Makefile.am
--- gst-plugins-base-1.14.5.old/gst-libs/gst/tag/Makefile.am	2018-03-23 20:44:42.000000000 -0000
+++ gst-plugins-base-1.14.5/gst-libs/gst/tag/Makefile.am	2020-01-25 02:40:57.378758050 -0000
@@ -1,11 +1,13 @@
 libgsttagincludedir = \
 	$(includedir)/gstreamer-@GST_API_VERSION@/gst/tag
 
+H := \#
+
 glib_enum_define = GST_TAG
 glib_gen_prefix = gst_tag
 glib_gen_basename = tag
 glib_gen_decl_banner=GST_TAG_API
-glib_gen_decl_include=\#include <gst/tag/tag-prelude.h>
+glib_gen_decl_include=$(H)include <gst/tag/tag-prelude.h>
 
 glib_enum_headers = tag.h gsttagdemux.h
 
diff -Naur gst-plugins-base-1.14.5.old/gst-libs/gst/tag/Makefile.in gst-plugins-base-1.14.5/gst-libs/gst/tag/Makefile.in
--- gst-plugins-base-1.14.5.old/gst-libs/gst/tag/Makefile.in	2019-05-29 16:33:24.000000000 -0000
+++ gst-plugins-base-1.14.5/gst-libs/gst/tag/Makefile.in	2020-01-25 02:42:51.689748130 -0000
@@ -17,12 +17,14 @@
 # these are the variables your Makefile.am should set
 # the example is based on the colorbalance interface
 
+H := \#
+
 #glib_enum_headers=$(colorbalance_headers)
 #glib_enum_define=GST_COLOR_BALANCE
 #glib_gen_prefix=gst_color_balance
 #glib_gen_basename=colorbalance
 #glib_gen_decl_banner=GST_EXPORT
-#glib_gen_decl_include=\#include <gst/foo/foo-prelude.h>
+#glib_gen_decl_include=$(H)include <gst/foo/foo-prelude.h>
 
 
 
@@ -680,12 +682,12 @@
 glib_gen_prefix = gst_tag
 glib_gen_basename = tag
 glib_gen_decl_banner = GST_TAG_API
-glib_gen_decl_include = \#include <gst/tag/tag-prelude.h>
+glib_gen_decl_include = $(H)include <gst/tag/tag-prelude.h>
 glib_enum_headers = tag.h gsttagdemux.h
 built_sources = tag-enumtypes.c
 built_headers = tag-enumtypes.h
 BUILT_SOURCES = $(built_sources) $(built_headers)
-enum_headers = $(foreach h,$(glib_enum_headers),\n\#include \"$(h)\")
+enum_headers = $(foreach h,$(glib_enum_headers),\n$(H)include \"$(h)\")
 libgsttaginclude_HEADERS = \
 	tag.h tag-prelude.h gsttagdemux.h gsttagmux.h xmpwriter.h
 
diff -Naur gst-plugins-base-1.14.5.old/gst-libs/gst/video/Makefile.am gst-plugins-base-1.14.5/gst-libs/gst/video/Makefile.am
--- gst-plugins-base-1.14.5.old/gst-libs/gst/video/Makefile.am	2018-05-09 06:29:09.000000000 -0000
+++ gst-plugins-base-1.14.5/gst-libs/gst/video/Makefile.am	2020-01-25 02:40:57.378758050 -0000
@@ -2,6 +2,8 @@
 ORC_SOURCE=video-orc
 include $(top_srcdir)/common/orc.mak
 
+H := \#
+
 glib_enum_headers = video.h video-format.h video-color.h video-info.h video-dither.h \
 			colorbalance.h navigation.h video-chroma.h video-tile.h video-converter.h \
 			video-resampler.h video-frame.h video-scaler.h
@@ -9,7 +11,7 @@
 glib_gen_prefix = gst_video
 glib_gen_basename = video
 glib_gen_decl_banner=GST_VIDEO_API
-glib_gen_decl_include=\#include <gst/video/video-prelude.h>
+glib_gen_decl_include=$(H)include <gst/video/video-prelude.h>
 
 built_sources = video-enumtypes.c
 built_headers = video-enumtypes.h
diff -Naur gst-plugins-base-1.14.5.old/gst-libs/gst/video/Makefile.in gst-plugins-base-1.14.5/gst-libs/gst/video/Makefile.in
--- gst-plugins-base-1.14.5.old/gst-libs/gst/video/Makefile.in	2019-05-29 16:33:24.000000000 -0000
+++ gst-plugins-base-1.14.5/gst-libs/gst/video/Makefile.in	2020-01-25 02:42:51.680748131 -0000
@@ -39,12 +39,14 @@
 # these are the variables your Makefile.am should set
 # the example is based on the colorbalance interface
 
+H := \#
+
 #glib_enum_headers=$(colorbalance_headers)
 #glib_enum_define=GST_COLOR_BALANCE
 #glib_gen_prefix=gst_color_balance
 #glib_gen_basename=colorbalance
 #glib_gen_decl_banner=GST_EXPORT
-#glib_gen_decl_include=\#include <gst/foo/foo-prelude.h>
+#glib_gen_decl_include=$(H)include <gst/foo/foo-prelude.h>
 
 
 
@@ -740,7 +742,7 @@
 glib_gen_prefix = gst_video
 glib_gen_basename = video
 glib_gen_decl_banner = GST_VIDEO_API
-glib_gen_decl_include = \#include <gst/video/video-prelude.h>
+glib_gen_decl_include = $(H)include <gst/video/video-prelude.h>
 built_sources = video-enumtypes.c
 built_headers = video-enumtypes.h
 lib_LTLIBRARIES = libgstvideo-@GST_API_VERSION@.la
@@ -821,7 +823,7 @@
 
 libgstvideo_@GST_API_VERSION@_la_LIBADD = $(GST_BASE_LIBS) $(GST_LIBS) $(ORC_LIBS) $(LIBM)
 libgstvideo_@GST_API_VERSION@_la_LDFLAGS = $(GST_LIB_LDFLAGS) $(GST_ALL_LDFLAGS) $(GST_LT_LDFLAGS)
-enum_headers = $(foreach h,$(glib_enum_headers),\n\#include \"$(h)\")
+enum_headers = $(foreach h,$(glib_enum_headers),\n$(H)include \"$(h)\")
 @HAVE_INTROSPECTION_TRUE@BUILT_GIRSOURCES = GstVideo-@GST_API_VERSION@.gir
 @HAVE_INTROSPECTION_TRUE@gir_headers = $(patsubst %,$(srcdir)/%, \
 @HAVE_INTROSPECTION_TRUE@	$(libgstvideo_@GST_API_VERSION@include_HEADERS)) \
